_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -1,21 +1,61 @@
 ---
 layout: default
-title: Amazon S3 source
+title: `s3` source
 parent: Sources
 grand_parent: Pipelines
 nav_order: 20
 ---
 
 # `s3` source
 
-The Amazon Simple Storage Service (Amazon S3) source plugin reads events from [S3](https://aws.amazon.com/s3/) objects. The following table describes options you can use to configure the `s3` source.
+`s3` is a source plugin that reads events from [Amazon Simple Storage Service (S3)](https://aws.amazon.com/s3/) objects. It requires an [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/) queue that receives [S3 Event Notifications](https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html). After SQS is configured, the `s3` source receives messages from SQS. When the SWS message indicates an S3 object was created, the `s3` source loads the objects than parses them using the configured [codec](#codec). You can also configure the `s3` source to use [Amazon S3 Select](https://docs.aws.amazon.com/AmazonS3/latest/userguide/selecting-content-from-objects.html) to parse S3 objects instead of Data Prepper.
```

Comment:
```suggestion
`s3` is a source plugin that reads events from [Amazon Simple Storage Service (S3)](https://aws.amazon.com/s3/) objects. It requires an [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/) queue that receives [S3 Event Notifications](https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html). After Amazon SQS is configured, the `s3` source receives messages from SQS. When the SWS message indicates an S3 object was created, the `s3` source loads the objects than parses them using the configured [codec](#codec). You can also configure the `s3` source to use [Amazon S3 Select](https://docs.aws.amazon.com/AmazonS3/latest/userguide/selecting-content-from-objects.html) to parse S3 objects instead of Data Prepper.
```

Reply:
Yarp. Good catch.